### PR TITLE
Event detail a pi

### PR DESCRIPTION
**Pull request by amansd25**

### CODE REVIEW

File: controllers/eventController.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const User = require('../models/userModel');
 const mongoose = require("mongoose");
 const ObjectId = mongoose.Types.ObjectId;
 const { otpgenerator } = require('../utils/emailService');
+const { error } = require('winston');
```
Comment (BharatShukla30) [inline controllers/eventController.js:10]: what is this for?

File: controllers/eventController.js [hunk 2]
```diff
@@ -156,6 +157,40 @@ exports.joinEvent = async (req, res) => {
     }
 }
 
+exports.eventDetails = async(req,res)=>{
+    const eventId = (req.params['eventId']);
+    console.log(eventId)
+    try {
+        activityLogger.info("event Id recived");
```
Comment (BharatShukla30) [inline controllers/eventController.js:164]: which event id is received? spelling issue with received

File: controllers/eventController.js [hunk 3]
```diff
@@ -156,6 +157,40 @@ exports.joinEvent = async (req, res) => {
...
+            attributes: ['eventid', 'eventname', 'description', 'starttime', 'endtime', 'location', 'multimedia'],
+            
+        });
+        if (!event) {
+            return res.status(404).json({ msg: 'Event not found' });
+        }
+           console.log(event)
+        // Format the response
+        const eventDetails = {
+            eventId: event.eventid,
```
Comment (BharatShukla30) [inline controllers/eventController.js:175]: user id and user's display picture should also been given

File: controllers/eventController.js [hunk 4]
```diff
@@ -156,6 +157,40 @@ exports.joinEvent = async (req, res) => {
...
+            title: event.eventname,
+            description: event.description,
+            date: event.starttime,
+            time: event.endtime,
+            location: event.location,
```
Comment (BharatShukla30) [inline controllers/eventController.js:180]: multimedia should also be there

**Review by BharatShukla30 — CHANGES_REQUESTED** [review]

File: controllers/eventController.js (hunk 1 above)
Comment (amansd25) [reply controllers/eventController.js:10]: it has the same working as activity logger and error logger so as we already have activity logger and error logger in our code so we don't need const { error } = require('winston');

File: controllers/eventController.js (hunk 2 above)
Comment (amansd25) [reply controllers/eventController.js:164]: the event which we call

File: controllers/eventController.js (hunk 4 above)
Comment (amansd25) [reply controllers/eventController.js:180]: added

**Pull request merged**